### ISSUE TITLE
The UI generated by  easygui is not readable with screenreader. 

### ISSUE THREAD

**Issue by hpkotian** [hpkotian/screenreader#1]
The UI generated by  easygui is not readable with screenreader. 
What is screenreader: 
https://axesslab.com/what-is-a-screen-reader/
What is application development guidelines: 
https://www.w3.org/WAI/standards-guidelines/mobile/#:~:text=User%20Agent%20Accessibility%20Guidelines%20(UAAG,Mobile%20Accessibility%20Examples%20from%20UAAG. 
Please work on it.
